File: OCH_Tao/ABC/A/331.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [M, D] = input[0].split(" ").map(x => parseInt(x, 10));
const [y, m, d] = input[1].split(" ").map(x => parseInt(x, 10));
if (M === m && D === d) {
  console.log(`${y + 1} 1 1`);
} else if (D === d) {
  console.log(`${y} ${m + 1} 1`);
} else {
  console.log(`${y} ${m} ${d + 1}`);
}

File: OCH_Tao/ABC/A/331.py
M,D = map(int,input().split())
YMD = list(map(int,input().split()))
if YMD[2]<D:
  YMD[2]+=1
  print(*YMD)
else:
  YMD[2]=1
  if YMD[1]<M:
    YMD[1]+=1
    print(*YMD)
  else:
    YMD[1]=1
    YMD[0]+=1
    print(*YMD)
